installation/aws/VmClarity.cfn

```diff
@@ -170,8 +182,12 @@ Resources:
                     AWS_INSTANCE_TYPE=${ScannerInstanceType}
                     SCANNER_KEY_PAIR_NAME=${KeyName}
                     AWS_JOB_IMAGE_ID=${JobImageID}
-                    DATABASE_DRIVER=LOCAL
-                    LOCAL_DB_PATH=/data/vmclarity.db
+                    DATABASE_DRIVER=POSTGRES
```

Comment:
Do we want to keep the support for sqlite?

Reply:
I believe so its super useful for local testing/development as well as for simple deployments.

Reply:
or do you mean in the cloud formation as a configuration option?

Reply:
done, you can now choose which DB backend to use, including an external one